src/main/scala/org/zalando/spearheads/innkeeper/services/RoutesService.scala

```diff
@@ -19,7 +19,7 @@ trait RoutesService {
              createdBy: String,
              createdAt: LocalDateTime = LocalDateTime.now()): Future[Result[RouteOut]]
 
-  def remove(id: Long): Future[Result[RouteOut]]
+  def remove(id: Long): Future[Result[Boolean]]
```

Comment:
I don't really like the move to Boolean here. If you want to distinguish between "DeletedSuccessfully" "NotDeleted", create a type with those two inhabitants